homeassistant/components/cover/ryobi_gdo.py

```diff
@@ -0,0 +1,285 @@
+"""
+Ryobi platform for the cover component.
+
+For more details about this platform, please refer to the documentation
+https://home-assistant.io/components/ryobiGDO/
+"""
+import logging
+import json
+import socket
+from datetime import timedelta
+import voluptuous as vol
+import requests
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.components.cover import (
+    CoverDevice, PLATFORM_SCHEMA, SUPPORT_OPEN, SUPPORT_CLOSE)
+from homeassistant.helpers.event import track_utc_time_change
+from homeassistant.const import (
+    CONF_DEVICE, CONF_USERNAME, CONF_PASSWORD,
+    STATE_UNKNOWN, STATE_CLOSED, STATE_OPEN, CONF_COVERS)
```

Comment:
The import should be grouped and ordered -> https://developers.home-assistant.io/docs/en/development_guidelines.html